lib/game_setup.rb

```diff
@@ -24,20 +23,15 @@ def welcome
     input = gets.chomp
 
       if input == "p" || input == "P"
-        self.start_game
+        setup
       elsif input == 'q' || input == "Q"
         exit
       else
         puts 'INVALID INPUT'
-        self.welcome
+        welcome
       end
   end
 
-  def start_game
```

Comment:
@m-mrcr  this was repetitive so I deleted it, fyi

Reply:
Yes, good. :)

Reply:
Happy about the work we've done. Merging pull request.